### PR TITLE
UTs for CloudAuditLogsSource validation

### PR DESCRIPTION
**Pull request by nlopezgi**
Fixes https://github.com/google/knative-gcp/issues/493

## Proposed Changes

Adding UTs for CloudAuditLogsSource validation

### CODE REVIEW

File: pkg/apis/events/v1alpha1/cloudauditlogssource_validation_test.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
+/*
+Copyright 2019 Google LLC
```
Comment (nachocano) [inline pkg/apis/events/v1alpha1/cloudauditlogssource_validation_test.go]: 2020

**Comment by nachocano** [comment]
Thanks for doing this @nlopezgi !!!
Just an update to the copyright, and it should be in!

File: pkg/apis/events/v1alpha1/cloudauditlogssource_validation_test.go (hunk 1 above)
Comment (nlopezgi) [reply pkg/apis/events/v1alpha1/cloudauditlogssource_validation_test.go]: done

**Comment by nlopezgi** [comment]
> Thanks for doing this @nlopezgi !!!
> Just an update to the copyright, and it should be in!

resolved, should be ready to go in.

**Comment by nachocano** [comment]
/lgtm
/approve

Thanks @nlopezgi !

**Pull request merged**